### PR TITLE
Update Evergreen

### PR DESCRIPTION
**Pull request by ShalokShalom**
Is this how updates are supposed to happen?

Do we have someone being subscribed to all the repos, to get notified of releases?

How do we handle this?
https://github.com/TorchedSammy/Evergreen.lxl/discussions/52

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -207,7 +207,7 @@
     },
     {
       "description": "Adds Treesitter syntax highlighting support",
-      "version": "0.1",
+      "version": "0.2",
       "remote": "https://github.com/TorchedSammy/Evergreen.lxl:c4058bfdedd27aee3b23319905b5df7b4ca52ef8",
```
Comment (jgmdev) [inline manifest.json:211]: Also the commit hash needs to be updated to that of the latest plugin version:

```suggestion
"remote": "https://github.com/TorchedSammy/Evergreen.lxl:e5e6ade897a6316fbe6fc610520d195e27737912",
```

**Comment by jgmdev** [comment]
> Is this how updates are supposed to happen?
> 
> Do we have someone being subscribed to all the repos, to get notified of releases?

Usually the plugin author would submit a PR to update the plugins repository to include latest version but in this case the PR hasn't been done even to lite-xl.

In any case feel free to monitor the plugin if you use it and submit new PR versions to this repository, I will happily merge it. I wonder if we can somehow automate this or at least have a global monitoring system that checks the plugin version available on the third party repositories against the version on this repo.

**Comment by ShalokShalom** [comment]
I am pretty sure CI can do this?

**Comment by jgmdev** [comment]
> I am pretty sure CI can do this?

I guess a script that parses the manifest.json and compares its pointed commits against that of the third party plugin main repository by using the GitHub api to check the latest release or commit of the pointed third party plugin on its master/main branch, or instead of using the API just cloning the related repositories and do the check directly (which could be slower but would make it more easy to support any non GitHub code hosting platform).

The CI should create a PR for review to reduce a bit the dangers of bad intentioned users that could initially provide a good plugin at first and then add bad stuff to it. This would at least be feasible for Lua plugins. For native binary plugins it would be harder to ensure security, maybe shipping a build system for Pragtical so that users by default would need to compile the plugin source and optionally opt for precompiled binaries which would be advertised as not secure. 

Anyways this could be subject for a separate issue ticket or a discussion for which I would need to enable them on this repository first or better yet a PR implementing it :)

**Comment by ShalokShalom** [comment]
I think it's easier to simply sign up to be notified for new releases, for now.
Its not like our ecosystem is so huge, that we automate that, yet.

**Comment by jgmdev** [comment]
> I think it's easier to simply sign up to be notified for new releases, for now.

Agree

**Pull request merged**